app/views/dashboard/progress.html.erb

```diff
@@ -26,6 +26,7 @@
          data-proposal-graph-success-label="<%= t("dashboard.progress.success") %>"
          data-proposal-success="<%= Setting["votes_for_proposal_success"] %>"
          data-proposal-resources-url="<%= proposal_dashboard_resources_path(proposal, format: :json) %>"
+         data-proposal-settings-url="<%= proposal_settings_path(format: :json) %>"
```

Comment:
Do you think it would work if we passed something like:
```
         data-enable-successful-proposals="<%= Setting["feature.dashboard.successful_proposals"] %>"
```
In a similar way we pass `Setting["votes_for_proposal_success"]`? That way we could simplify the code and remove the controller :thinking:.